doc/en/user/source/community/features-templating/directives.rst

```diff
@@ -709,14 +709,20 @@ The :code:`flat_output` will act in the following way:
 
 JSON-LD
 """"""""
-A JSON-LD template can be defined as a GeoJSON template since it is a JSON based output as well. However it needs to have a :code:`@context` attribute, object or array at the beginning of it in orther to conform to the standard.
-To accomplish this requirement it is possible to specify the :code:`@context` as an :code:`option` in the template, like in the following one:
+A JSON-LD template can be defined as a GeoJSON template since it is a JSON based output as well. However it needs to have a :code:`@context` attribute, object or array at the beginning of it in orther to conform to the standard. Moreover each JSON Object must have an :code:`@type` defining a type through a vocabulary term.
```

Comment:
```suggestion
A JSON-LD template can be defined as a GeoJSON template since it is a JSON based output as well. However it needs to have a :code:`@context` attribute, object or array at the beginning of it in order to conform to the standard. Moreover each JSON Object must have an :code:`@type` defining a type through a vocabulary term.
```

Reply:
fixed